lib/parser.js

```diff
@@ -8,7 +8,7 @@ var suffixes = [
   'PE', 'PHD', 'SJ', 'SR', 'V', 'VI', 'VP', 'V', 'I', 'VI', 'VII', 'VIII',
   'CNP', 'CPA', 'DDS', 'DMin', 'DMA ', 'DMus', 'OD', 'DO', 'PharmD', 'PhD',
   'PsyD', 'DVM', 'EI', 'EIT', 'Esq', 'JD', 'LLS', 'LP', 'LPN',
-  'MD', 'PE', 'RA', 'RLA', 'RLS', 'RN', 'SE'
+  'MD', 'PE', 'RA', 'RLA', 'RLS', 'RN', 'SE', 'Esquire', 'ESQUIRE'
```

Comment:
We're already using case-insensitive comparisons [here](https://github.com/mklaber/node-another-name-parser/blob/master/lib/util.js#L14). Can you remove one of these?